tools/data-collect/rs-data-collect.cpp

```diff
@@ -124,7 +124,7 @@ void data_collector::save_data_to_file(const string& out_filename)
 
     for (const auto& elem : data_collection)
     {
-        csv << "\n\nStream Type,Index,F#,HW Timestamp (ms),Host Timestamp(ms)"
+        csv << "\n\nStream Type,Index,F#,HW Timestamp (ms),Host Timestamp(ms), SerialNumber"
```

Comment:
The `SerialNumber` attribute to be moved into the header (Lines 121-123 : csv << get_profile_description(elem);)